ios/Runner/Runner.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
 	<key>com.apple.developer.applesignin</key>
```

Comment:
Is this file just for development things? Or why is the environment "development"?

Reply:
I have no idea tbh, this is what is added by XCode after I enable push notification. When I remove this flag from this file, push notifications are disabled

Reply:
Sounds good to me